sunpy/io/special/asdf/schemas/sunpy.org/sunpy/coordinates/frames/helioprojective-1.0.0.yaml

```diff
@@ -31,8 +31,10 @@ examples:
                     wrap_angle: !<tag:astropy.org:astropy/coordinates/angle-1.0.0> {unit: !unit/unit-1.0.0 deg,
                       value: 180.0}
                 type: SphericalRepresentation
-              frame_attributes: {obstime: !time/time-1.1.0 '2011-01-01T01:01:01.000'}
-            obstime: !time/time-1.1.0 2011-01-01T01:01:01.000
+              frame_attributes:
+                obstime: !time/time-1.1.0 '2011-01-01T01:01:01.000'
+                rsun: !unit/quantity-1.1.0 {unit: !unit/unit-1.0.0 km, value: 695700.0}
+            obstime: !time/time-1.1.0 '2011-01-01T01:01:01.000'
```

Comment:
I don't know much about these, `.yaml` files, but just wanted to check that it was intended to create new ones for carrington and stonyhurst, but modify an existing one for helioprojective?

Reply:
The schema for `Helioprojective` itself doesn't change, but I updated the example it shows the breakdown for the `observer` attribute (`HeliographicStonyhurst`).  I guess I neglected to update the example for `Heliocentric`.